app/javascript/controllers/object_reporter.js

```diff
@@ -0,0 +1,116 @@
+import bootstrap from "bootstrap/dist/js/bootstrap";
+import { Controller } from "@hotwired/stimulus";
+import { DataTable, exportCSV } from "simple-datatables";
+
+export default class extends Controller {
+  static targets = ["columnSelector"];
+  static values = {
+    dataUrl: String,
+    columnModel: Array,
+  };
+
+  connect() {
+    if (this.table !== undefined) return; // Prevent negative side effects if hook is called more than once per page load
+
+    this.table = new DataTable("#objectsTable", {
+      columns: this.transformColumns(),
+      searchable: true,
+      sortable: true,
+      scrollY: "70vh",
+      paging: false,
```

Comment:
Approximate the former look & feel.

Reply:
is there a design by Astrid or other UI guidance?

Reply:
There is an old design that pre-dates Astrid, I believe, but for the current PR I've aimed for "as close to parity as we can get using standard simple-datatables config"